cmd/ci-images-mirror/main.go

```diff
@@ -86,6 +89,27 @@ func newOpts() *options {
 	return opts
 }
 
+func (o *options) loadConfig(caller string) error {
+	o.configMutex.Lock()
+	defer o.configMutex.Unlock()
+	now := time.Now()
+	if o.configFile == "" {
+		return fmt.Errorf("cannot load config when the config file is empty")
+	}
+	if o.config != nil && now.Before(o.configLastModifiedAt.Add(30*time.Minute)) {
```

Comment:
Any particular reason for checking before 30 min specifically?

Reply:
I think anything that is smaller than `1h` is good enough.
`1h` is the interval to run the sync on ci-sup and art images.